docs/fundamentals/code-analysis/style-rules/ide0004.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "IDE0004: Remove unnecessary cast"
+description: "Learn about code analysis rule IDE0004: Remove unnecessary cast"
+ms.date: 09/30/2020
+ms.topic: reference
+f1_keywords:
+- IDE0004
+- cast
+- unnecessary
```

Comment:
F1 keywords shouldn't have any duplicates.
I would keep only the diagnostic ID and editor config options (if any)

Reply:
Ah got you, thanks.